kubernetes/values.yaml

```diff
@@ -53,9 +66,12 @@ config:
   db:
     user: kemal
     password: kemal
-    host: invidious-postgresql
+    #-- Leave empty if you use embeded postgresql server
+    host:
     port: 5432
     dbname: invidious
   full_refresh: false
   https_only: false
   domain:
+  #-- Generate a key using `pwgen 20 1` for example
+  hmac_key: CHANGEME
```

Comment:
`CHANGE_ME!!` is checked for in the code so it might be better to use that
https://github.com/iv-org/invidious/blob/701f03d5e88a8f3c92575965e39b8f96b59f71f9/src/invidious/config.cr#L210

Reply:
Oh yes, indeed !